taxi/views.py

```diff
@@ -1,3 +1,16 @@
 from django.shortcuts import render
 
-# Create your views here.
+from django.http import HttpResponse, HttpRequest
+from .models import Driver, Manufacturer, Car
```

Comment:
fix the imports. python modules first, then installed modules, then your own and each group should be separated by blank line